config/routes.rb

```diff
@@ -26,16 +26,22 @@
         get "confirm", on: :member
         put "record", on: :member
 
+        post "consent", on: :member
```

Comment:
Does `VaccinationsController#consent` create anything? Or is this going to be changed in the future?

Reply:
No, I wasn't sure what to use between `POST/PATCH/PUT` since nothing is being mutated. Thoughts?

Reply:
`POST` is probably closer in intent to "this will eventually create a new consent, just not on this specific action" :thinking:

Reply:
Posted two comments before I remembered; yes, the "No" route should create a consent.